site/content/en/docs/Installation/helm.md

```diff
@@ -130,13 +130,13 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.resources`                       | Controller resource requests/limit                                                              | `{}`                   |
 | `agones.controller.generateTLS`                     | Set to true to generate TLS certificates or false to provide your own certificates in `certs/*` | `true`                 |
 | `agones.ping.install`                               | Whether to install the [ping service][ping]                                                     | `true`                 |
-| `agones.ping.replicas`                              | The number of replicas to run in the deployment                                                 | `2`                    | 
-| `agones.ping.http.expose`                           | Expose the http ping service via a Service                                                      | `true`                 | 
-| `agones.ping.http.response`                         | The string response returned from the http service                                              | `ok`                   | 
+| `agones.ping.replicas`                              | The number of replicas to run in the deployment                                                 | `2`                    |
```

Comment:
These new configuration params need to be moved to the "new configuration features" section below as well, so they are hidden when I work on CD for the site this week.

Reply:
those *ping* changes are not part of my change. I think we should move them in separate commit.

Reply:
ignore me. they are trailing line removals.